src/server/db/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "development": {
+    "username": "user",
+    "password": "password",
+    "database": "leket",
+    "host": "localhost",
+    "port": 5433,
+    "dialect": "postgres"
+  },
+  "test": {
+    "username": "root",
+    "password": null,
+    "database": "leket",
+    "host": "192.168.99.100",
+    "port": 5433,
+    "dialect": "postgres"
+  },
+  "production": {
+    "use_env_variable": "DATABASE_URL"
```

Comment:
The fact that you can use `process.env.HOST` - it keeps the configuration in this file instead of "leaking" it (using the flag) to the `index.js` file